docs/servicenow.md

```diff
@@ -0,0 +1,173 @@
+# ticketutil.servicenow
+
+This document contains information on the methods available when working
+with a ServiceNow object. A list of the ServiceNow fields that have been
+tested when creating and editing tickets is included. Because each
```

Comment:
I would start a new paragraph from "Because each ...". I would also add "Note: " before "Because...".

IMHO it would be easier to  read and feel little bit easier also :-) However this is just my feeling and feel free to ignore this.